test/hga/test_utilities.py

```diff
@@ -12,6 +12,20 @@
 from pyproj import Transformer
 
 
+def get_test_granule_id(environment: Environment, production_granule_id: str,
+                        non_prod_granule_id: str) -> str:
+    """ Select either the production or non-production granule concept ID
+        based on the environment against which the tests are being run.
+
+    """
+    if environment == Environment.PROD:
+        granule_id = production_granule_id
+    else:
+        granule_id = non_prod_granule_id
```

Comment:
I could compress the lines above into a Python ternary, but I don't think those are as legible as in something like JavaScript/TypeScript.

Reply:
I prefer clarity over condensing every time.  This is clear to me.